### DISCUSSION TITLE
Typewriter mode should have smoother transitions between individual characters

### DISCUSSION THREAD

**Discussion by raccoonhands1** [monkeytypegame/monkeytype#4602] [Ideas]
its too hard to follow individual characters if you go 90+ WPM

**Comment by Miodec** [comment]
Typewriter mode?

**Reply by Warrobot10** [reply]
I think they are talking about the sound on press sound “typewriter”

**Comment by raccoonhands1** [comment]
> Typewriter mode?

For clarification, I'm referring to the alternate layout in the funbox

**Reply by Miodec** [reply]
Sorry, which funbox?
